server/api/offline_results.py

```diff
@@ -2,6 +2,7 @@
 from typing import List, Optional
 from flask import jsonify, request
 from werkzeug.exceptions import BadRequest, Conflict
+from sqlalchemy import func
```

Comment:
I'm a little confused about these `import` statements in a few files that don't seem to come with any other changes. What's going on there?

Reply:
Oh right, should have explained that. Previously, there was a wildcard import `import * from sqlalchemy` in models.py, so all of these names like `func` were imported implicitly and then exported to every file that imported models.py. That caused a conflict when importing the variable `engine` from database.py, since sqlalchemy also has a name `engine` that's something different. I decided to get rid of the wildcard import and be more explicit to avoid issues like this in the future.